qiskit-qir/setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = qiskit-qir
+version = 0.0.1
+author = Microsoft
+author_email = que-contacts@microsoft.com
+description = Qiskit to QIR translator
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/microsoft/qdk-python
+project_urls =
+    Bug Tracker = https://github.com/microsoft/qdk-python
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: Microsoft
+    Operating System :: MacOS
+    Operating System :: Unix
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.7
+install_requires =
+    qiskit-terra>=0.19.2
+    pyqir-generator@https://github.com/qir-alliance/pyqir/releases/download/v0.2.0a1/pyqir_generator-0.2.0a1-cp36-abi3-macosx_10_7_x86_64.whl ; platform_system=="Darwin"
```

Comment:
Mentioning https://github.com/qir-alliance/pyqir/issues/34 to track that dependency.